### PR TITLE
Aave stETH/ETH product header

### PR DESCRIPTION
**Pull request by marcinciarka**
# [Aave stETH/ETH product header](https://app.shortcut.com/oazo-apps/story/4806/product-header)

## Changes 👷‍♀️
- TBD
  
## How to test 🧪
- TBD

### CODE REVIEW

File: components/vault/VaultHeadlineDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import { Box, Text } from 'theme-ui'
+
+export interface HeadlineDetailsProp {
+  label: string
+  value: string | number
+  sub?: string
+  subColor?: string
+}
+
+export function VaultHeadlineDetails({ label, value, sub, subColor }: HeadlineDetailsProp) {
```
Comment (sennett) [inline components/vault/VaultHeadlineDetails.tsx:11]: NIce

File: components/vault/EarnVaultHeadline.tsx [hunk 2]
```diff
@@ -1,50 +1,7 @@
...
-        }}
-      >
-        {details.map((details) => (
-          <VaultHeadlineDetails {...details} key={details.label} />
-        ))}
-      </Flex>
-    </Flex>
-  )
+export function EarnVaultHeadline({ header, token, details }: VaultHeadlineProps) {
+  return <VaultHeadline header={header} token={token} details={details} />
```
[the remark this answers is not in the document]
Comment (marcinciarka) [reply components/vault/EarnVaultHeadline.tsx:6]: That goes to the tech debt - sorting out the header/headline/heading components.

File: blockchain/aavePrepareAaveTotalValueLocked.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+          'ETH',
+        ) // total stable debt in WETH
+        const WETH_totalVariableDebt = amountFromWei(
+          new BigNumber(ETH_reserveData.totalVariableDebt),
+          'ETH',
+        ) // total variable debt in WETH
+
+        const STETH_USDC_availableLiquidity = STETH_availableLiquidity.times(STETH_USDC_price) // available liquidity in STETH in USDC
+        const USDC_WETH_debt = WETH_totalStableDebt.plus(WETH_totalVariableDebt).times(
+          ETH_USDC_price,
```
[the remark this answers is not in the document]
Comment (marcinciarka) [reply blockchain/aavePrepareAaveTotalValueLocked.ts]: Done.

File: components/vault/VaultHeadlineDetails.tsx (hunk 1 above)
Comment (marcinciarka) [reply components/vault/VaultHeadlineDetails.tsx:11]: 🥃

**Review by BackToTheCode — APPROVED** [review]
Really great. You've clean up a lot of components too.

**Pull request merged**